crates/jxl-render/src/lib.rs

```diff
@@ -116,7 +116,7 @@ impl RenderContext {
 }
 
 impl RenderContext {
-    fn render_by_index(&mut self, index: usize, image_region: Option<Region>) -> Result<()> {
+    fn render_by_index(&mut self, index: usize, image_region: Option<Region>) -> Result<Region> {
```

Comment:
This method returns the frame region for intersection with image region?

Reply:
It's needed to correctly crop the rendered image, which is in frame coordinate.